pom.xml

```diff
@@ -356,7 +381,7 @@
             <goals>
               <goal>apply</goal>
             </goals>
-            <phase>compile</phase>
+            <phase>generate-sources</phase>
```

Comment:
(Minor/Question)

Shouldn't spotless run on "process-sources", i.e. after "generate-sources"?

Reply:
You're right, corrected